mcf-browser/src/back-end/parse-tmcf.js

```diff
@@ -0,0 +1,232 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as csv from 'csvtojson';
+import {shouldReadLine} from './utils.js';
+
+/**
+ * Returns the string following '->' in  a given string. Used for getting csv
+ * column name when filling in tmcf with values from csv.
+ * Ex:   C:SomeDataset->GeoId would return 'GeoId'
+ * @param {string} propValue The string to look for a column name in.
+ * @return {string|null} The column name that comes after '->'.
+ */
+function getArrowId(propValue) {
+  if (propValue.includes('->')) {
+    return propValue.split('->')[1];
+  }
+  return null;
+}
```

Comment:
Similar to getEntityId() wonder why you do not return the whole C:SomeDataSet->GeoId, and then replace that with the CSV cell value?